keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -861,12 +865,19 @@ enhancement:
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
   the enablement)?**
-  Yes.
 
-  In Kubernetes 1.19 and 1.20, registration of the
-  `CSIStorageCapacity` type was controlled by the feature gate. In
-  1.21, the type will always be enabled in the v1beta1 API
-  group. Depending on the combination of Kubernetes release and
+  Yes, by disabling it in the CSI driver deployment:
+  `CSIDriver.StorageCapacity=false` causes kube-scheduler to ignore storage
+  capacity for the driver. In addition, external-provisioner can be deployed so
+  that it does not publish capacity information (`--enable-capacity=false`).
+
+  Downgrading to a previous Kubernetes release may also disable the feature or
+  allow disabling it via a feature gate: In Kubernetes 1.19 and 1.20,
+  registration of the `CSIStorageCapacity` type was controlled by the feature
+  gate. In 1.21, the type will always be enabled in the v1beta1 API group. In
+  1.24, the type is always enabled in the v1 API unconditionally.
+
+  Depending on the combination of Kubernetes release and
```

Comment:
Two more comments (can't comment on the corresponding lines):

L908 - were those test created?
L995 - can you fill this in (or discuss the potential SLO values).

Reply:
> L908 - were those test created?

Yes. I've added two example to the test.

> L995 - can you fill this in (or discuss the potential SLO values).

I think our conclusion is that this depends too much on the specific CSI driver and how it is deployed to say more about SLOs in a KEP. I've replaced that sentence with a section explaining that thinking. I hope this is good enough because I can't think of much that we can do upstream about this.

Reply:
That sounds fine - thanks!